src/table/hooks/use-column-widths.ts

```diff
@@ -79,14 +79,18 @@ export const getColumnWidths = (
 
 const useColumnWidths = (
   columns: Column[],
-  tableWidth: number
+  tableWidth: number,
+  { head, body }: TableStyling
 ): [number[], React.Dispatch<React.SetStateAction<number[]>>] => {
-  const measureTextMethods = useMeasureText('13px', 'Arial');
-  // TODO use actual font size (and font eventually)
-  const [columnWidths, setColumnWidths] = useState(getColumnWidths(columns, tableWidth, measureTextMethods));
+  const measureHeadLabel = useMeasureText(head.fontSize, head.fontFamily).measureText;
+  const { measureText, estimateWidth } = useMeasureText(body.fontSize, body.fontFamily);
+  const getHugWidth = (headLabel: string, totalsLabel: string, glyphCount: number) =>
+    Math.max(measureHeadLabel(headLabel), measureText(totalsLabel), estimateWidth(glyphCount));
```

Comment:
"padding", "border" and that menu button will also occupy space so that should be accounted for when measuring the require width. I do think that is out-of-scope for this PR but just an FYI.

Reply:
yes I mention it in the description